docker/Dockerfile.bridge

```diff
@@ -35,7 +35,9 @@ COPY ./rpc ./rpc
 RUN cargo build -p trin -p portal-bridge --release
 
 # final base
-FROM ubuntu:22.04
+FROM ubuntu:20.04
```

Comment:
I think it's also worth a comment mentioning why we're on an old ubuntu in both dockerfiles